code/resources/public/ui/config/config.json

```diff
@@ -7,7 +7,7 @@
   "eula": null,
   "terms-and-conditions": null,
   "mapbox-access-token": null,
-  "stripe": null,
+  "stripe": "xxx",
```

Comment:
@0xbase12 is this change made on purpose or committed by mistake ?

Reply:
mistake